### ISSUE TITLE
Future of control directory

### ISSUE THREAD

**Issue by mauricefallon** [openhumanoids/oh-distro#50]
@rdeits has been pulling the InstQP into drake and making it C++. Is there a future for the 'control' directory? A lot of its is stale matlab and quite a bit of it hasn't been touched since the Trials.

We had talked about this briefly at the DRC. @rdeits :  Is it possible to simple ditch it going forward?

For example @MarcoCar has started (in his branch) a separate directory for motion planning (using Drake) which never had a specific home.

**Comment by rdeits** [comment]
Yes, my goal is to get rid of the control folder and move all of the useful parts of it into drake. I'm currently stuck on https://github.com/RobotLocomotion/drake/issues/1672 but I think most of the scary controller reorganization is done, and the rest is just cleaning up interfaces and putting it all together.
